package.json

```diff
@@ -111,6 +111,8 @@
     "proxy:build:release": "cd proxy && cargo build --release --all-features --all-targets",
     "proxy:start": "cd proxy && cargo run -- --registry=emulator",
     "proxy:start:test": "cd proxy && cargo run -- --registry=emulator --test",
+    "proxy:start:ffnet": "cd proxy && cargo run -- --registry=rpc.ff.radicle.network",
+    "proxy:start:devnet": "cd proxy && cargo run -- --registry=35.241.138.91",
```

Comment:
Can we put this behind a DNS entry, like `rpc.devnet.radicle.network`?

Reply:
https://github.com/radicle-dev/infra/issues/122